pkg/cli/interactive_tests/test_demo_cli_integration.tcl.disabled

```diff
@@ -47,28 +47,37 @@ eexpect "does not exist"
 eexpect "defaultdb>"
 send "\\q\r"
 eexpect eof
+
+spawn $argv sql --no-line-editor -u root -d cluster:demo-tenant
```

Comment:
it seems like this test file is being skipped. should we re-enable it?

there's a lot of other disabled ones too. perhaps we can check that they don't regress with this change, even if they are not re-enabled right now.

Reply:
Good call. I'll check them manually in this PR, then send another PR separately to re-enable them.